turbo.json

```diff
@@ -5,6 +5,10 @@
       "outputs": ["dist/**", "storybook-static/**"],
       "dependsOn": ["^build"]
     },
+    "check-exports": {
+      "cache": false,
+      "dependsOn": ["^build"]
```

Comment:
This is basically saying wait for the `build` step to complete before running `check-exports`. I looked into turbo a bit, pretty cool. Was your intention to eventually use it in Cambly-Frontend? Was CI really slow for Syntax?

Reply:
@williamnewman94 yes, that's correct. As for using `turbo` in `Cambly-Frontend`, I think that would only make sense if we move more into a monorepo setup with multiple packages (not something I immediately would recommend for us)